library-best-practices/JLBP-5.md

```diff
@@ -23,9 +23,6 @@ Example 1: Guava's main artifact ID is `guava`, but from versions 13.0 to 17.0,
   `guava-jdk5` with `guava` because the artifact names are different.
   When Guava classes overlapped in the two jars, users suffered from 
   runtime errors resulting from classes and methods not being found.
-  The only Google Cloud library propagating
```

Comment:
I think the example is helpful; I think we should just remove the incorrect claim that it is the only case.

Reply:
I though that's what I did.

Reply:
Well, I meant we should keep the example of the example (google-api-client depending on guava-jdp5)

Reply:
I don't think that helps the reader very much. It's quite specific and the general description with guava-jdk5 is clear enough.